doc/amazon_sagemaker_operators_for_kubernetes_jobs.rst

```diff
@@ -304,6 +304,135 @@ job stops or completes.
+            Value:          5
+          Collection Name:  losses
+          Collection Parameters:
+            Name:           save_interval"
+            Value:          500
+          Collection Name:  average_shap
+          Collection Parameters:
+            Name:           save_interval
+            Value:          5
+          Collection Name:  metrics
+          Collection Parameters:
+            Name:      save_interval
+            Value:     5
+        s3OutputPath:  s3://my-bucket/sagemaker/xgboost-mnist/xgboost-debugger/
+      Debug Rule Configurations:
+        Rule Configuration Name:  LossNotDecreasing
+        Rule Evaluator Image:     895741380848.dkr.ecr.us-west-2.amazonaws.com/sagemaker-debugger-rules:latest
+        Rule Parameters:
+          Name:   collection_names
+          Value:  metrics
+          Name:   num_steps
+          Value:  10
+          Name:   rule_to_invoke
+          Value:  LossNotDecreasing
+      Hyper Parameters:
+        Name:   max_depth
+        Value:  5
+        Name:   eta
+        Value:  0.2
+        Name:   gamma
+        Value:  4
+        Name:   min_child_weight
+        Value:  6
+        Name:   silent
+        Value:  0
+        Name:   objective
+        Value:  reg:squarederror
+        Name:   subsample
+        Value:  0.7
+        Name:   num_round
+        Value:  51
+      Input Data Config:
+        Channel Name:      train
+        Compression Type:  None
+        Content Type:      libsvm
+        Data Source:
+          s3DataSource:
+            s3DataDistributionType:  FullyReplicated
+            s3DataType:              S3Prefix
+            s3Uri:                   s3://my-bucket/sagemaker/xgboost-mnist/xgboost-debugger/train
+        Channel Name:                validation
+        Compression Type:            None
+        Content Type:                libsvm
+        Data Source:
+          s3DataSource:
+            s3DataDistributionType:  FullyReplicated
+            s3DataType:              S3Prefix
+            s3Uri:                   s3://my-bucket/sagemaker/xgboost-mnist/xgboost-debugger/validation
+      Output Data Config:
+        s3OutputPath:  s3://my-bucket/sagemaker/xgboost-mnist/xgboost-debugger/
+      Region:          us-west-2
+      Resource Config:
+        Instance Count:     1
+        Instance Type:      ml.m4.xlarge
+        Volume Size In GB:  5
+      Role Arn:             arn:aws:iam::1234567890:role/service-role/AmazonSageMaker-ExecutionRole
+      Stopping Condition:
+        Max Runtime In Seconds:  86400
+      Tags:
+        Key:              tagKey
+        Value:            tagValue
+      Training Job Name:  xgboost-mnist-debugger-8fe3799e68dd11ea84231260529a8dc9
+    Status:
+      Cloud Watch Log URL:  https://us-west-2.console.aws.amazon.com/cloudwatch/home?region=us-west-2#logStream:group=/aws/sagemaker/TrainingJobs;prefix=xgboost-mnist-debugger-8fe3799e68dd11ea84231260529a8dc9;streamFilter=typeLogStreamPrefix
```

Comment:
I don't think we need the full spec in this example. Perhaps just the Status field?

Reply:
I will still want to have full description so that even `spec` is clear to customer.